.gitignore

```diff
@@ -454,6 +454,9 @@ $RECYCLE.BIN/
 !.vscode/extensions.json
 
 
+<<<<<<< HEAD
```

Comment:
Do we need this? Think it can be excluded

Reply:
The line is in group gitignore, so not sure if someone else needs it.